docs/content/preview/drivers-orms/smart-drivers.md

```diff
@@ -22,6 +22,7 @@ Yugabyte has developed the following smart drivers for YSQL, available as open s
 | [YugabyteDB PGX Driver for Go](https://github.com/yugabyte/pgx) | jackc/pgx | [Documentation](../go/yb-pgx/) |
 | [YugabyteDB Psycopg2 Driver for Python](https://github.com/yugabyte/psycopg2) | PostgreSQL psycopg2 | [Documentation](../python/yugabyte-psycopg2/) |
 | [YugabyteDB node-postgres Driver for Node.js](https://github.com/yugabyte/node-postgres) | node-postgres | [Documentation](../nodejs/yugabyte-node-driver/) |
+| [YugabyteDB Npgsql Driver for C#](https://github.com/yugabyte/npgsql) | PostgreSQL Npgsql Driver | [Documentation](../csharp/ysql/) |
```

Comment:
correct github link? readme doesn't mention YB or "smart driver" at all

Reply:
Its a forked repo from npgsql. Not sure if they did this to some other projects too. @Sfurti-yb - I guess you must rework the readme like the Go driver : https://github.com/yugabyte/pgx